src/Avalonia.Controls/Primitives/Popup.cs

```diff
@@ -492,5 +470,54 @@ public void Dispose()
                 _owner._ignoreIsOpenChanged = false;
             }
         }
+
+        // For some reason when PopupRoot.Content is bound directly to Child weird stuff happens 
```

Comment:
Hmm, what weird stuff?

Reply:
Yes, for some reason the content gets stripped of all of its children.

Reply:
I've initially added `PopupContentHost` to debug the visual children disappearance, but it has suddenly stopped to be a problem, so I've left it as is.

Reply:
I think I will need a class like that for overlay popups, so I think we should keep it.